spec/models/garden_plant_spec.rb

```diff
@@ -0,0 +1,12 @@
+RSpec.describe GardenPlant, :type => :model do 
+  describe "garden creation by model" do 
+    context "garden association" do 
+      it { should belong_to(:user) }
+      it { should belong_to(:plant) }
+    end
+
+    context "garden validations" do 
+      it { should validate_presence_of(:user) }
+    end
+  end
+end
```

Comment:
This table needs no tests, it is simply a join table that allows us to make plants and users belong to eachother. So you can remove this.

Reply:
On further thought, I think this place will be the best place to test the association between a user and the plants in its garden.